### PR TITLE
atomist-cli 1.6.1

### PR DESCRIPTION
**Pull request by cdupuis**
Created by @atomist-bot.

### CODE REVIEW

File: Formula/atomist-cli.rb [hunk 1]
```diff
@@ -3,14 +3,8 @@
 class AtomistCli < Formula
   desc "The Atomist CLI"
   homepage "https://github.com/atomist/cli#readme"
-  url "https://registry.npmjs.org/@atomist/cli/-/@atomist/cli-1.6.0.tgz"
-  sha256 "0c00e070c8525df747676ea30241c772631c622664b26146f313fe1019778adb"
-
-  bottle do
```
Comment (Moisan) [inline Formula/atomist-cli.rb]: Please don't remove the `bottle` block, our merging process will update it automatically.

**Review by Moisan — CHANGES_REQUESTED** [review]

File: Formula/atomist-cli.rb (hunk 1 above)
Comment (ddgenome) [reply Formula/atomist-cli.rb]: Doesn't it also automatically add it if it is not there?

Comment (fxcoudert) [reply Formula/atomist-cli.rb]: It does, most of the time. But let's keep the changes to the minimal, so please restore it. (and squash into a single commit)

Comment (ddgenome) [reply Formula/atomist-cli.rb]: Okay. We just figured having no information was better than wrong information.

**Comment by fxcoudert** [comment]
Thanks! Merging.

(There won't be wrong information, because our merging process will include a second commit that updates the bottle information.)

**Comment by ddgenome** [comment]
I understand that ultimately master will not contain the wrong information, but the PR _does_ as the bottle reference in the PR point to the previous version.

**Pull request closed without merging**